app/models/review.rb

```diff
@@ -9,26 +9,24 @@ class Review < ActiveRecord::Base
       :review_date,
       :review_type
 
-  validates :review_type, :presence => true,
-            :inclusion => { :in => REVIEW_TYPES }
-  validates :associate_consultant_id, :presence => true
-  validates :associate_consultant_id, :uniqueness => {:scope => [:review_type]}
-  validates :feedback_deadline, :presence => true
-  validates :review_date, :presence => true
+  validates :review_type, presence: true, inclusion: { :in => REVIEW_TYPES }
+  validates :associate_consultant_id, presence: true
+  validates :associate_consultant_id, uniqueness: { scope: [:review_type]}
+  validates :feedback_deadline, presence: true
+  validates :review_date, presence: true
 
   belongs_to :associate_consultant
-  has_one    :reviewee, through: :associate_consultant, source: :user
 
-  has_many   :feedbacks,        :dependent => :destroy
-  has_many   :self_assessments, :dependent => :destroy
-  has_many   :invitations,      :dependent => :destroy
+  has_many   :feedbacks,        dependent: :destroy
+  has_many   :self_assessments, dependent: :destroy
+  has_many   :invitations,      dependent: :destroy
+  has_one    :reviewee, through: :associate_consultant, source: :user
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.